src/app/garfile/amg/checkin/pane.njk

```diff
@@ -60,7 +60,7 @@
 <form action="/garfile/amg/checkin" method="post" class="form">
   <h2 class="govuk-heading-m">{{__('heading_submit_gar')}}</h2>
   <p class="govuk-body-m">{{__('submit_gar_caption')}}</p>
-  <input id='review-submit' type="submit" {% if garfile.status.name != 'StatusCheckComplete' %}disabled{% endif %} class="govuk-button" value="{{__('amg_check_form_submit')}}">
+  <input id='review-submit' type="submit" {% 'disabled' if garfile.status.name != 'StatusCheckComplete' %} class="govuk-button" value="{{__('amg_check_form_submit')}}">
```

Comment:
Question: Is the `{% 'disabled' if garfile.status.name != 'StatusCheckComplete' %}` a valid **if expression**?

I presume it works.

- From the [docs](https://mozilla.github.io/nunjucks/templating.html#if-expression), it seems to be `{{ "true" if foo else "false" }}`.
- Also done in the departure status variant.

Reply:
yes it does work, and  I didn't want an else condition